### PR TITLE
Picture marker symbol offset

### PR DESCRIPTION
**Pull request by willcrick**
Added offsets to the Pin picture marker symbols so that their base aligns with the point geometry that they are showing.

### CODE REVIEW

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java [hunk 1]
```diff
@@ -35,168 +36,185 @@
...
+    mMapView.getGraphicsOverlays().add(graphicOverlay);
+
+    //Add graphics using different types of picture marker symbols
+
+    //[DocRef: Name=Picture Marker Symbol URL, Category=Fundamentals, Topic=Symbols and Renderers]
+    //Create a picture marker symbol from a URL resource
+    //When using a URL, you need to call load to fetch the remote resource
+    final PictureMarkerSymbol campsiteSymbol = new PictureMarkerSymbol(
+        "http://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae");
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
```
Comment (doneill) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: @willcrick You should split this comment after the comma to the next line comment.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java [hunk 2]
```diff
@@ -35,168 +36,185 @@
...
+        Graphic campsiteGraphic = new Graphic(campsitePoint, campsiteSymbol);
+        graphicOverlay.getGraphics().add(campsiteGraphic);
+      }
+    });
+
+    //[DocRef: Name=Picture Marker Symbol Drawable-android, Category=Fundamentals, Topic=Symbols and Renderers]
+    //Create a picture marker symbol from an app resource
+    BitmapDrawable pinStarBlueDrawable = (BitmapDrawable) ContextCompat.getDrawable(this, R.drawable.pin_star_blue);
+    final PictureMarkerSymbol pinStarBlueSymbol = new PictureMarkerSymbol(pinStarBlueDrawable);
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
```
Comment (doneill) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: @willcrick You should split this comment after the comma to the next line comment.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java [hunk 3]
```diff
@@ -35,168 +36,185 @@
...
+    });
+
+    //Check sample has access to external storage
+    if (saveResourceToExternalStorage()) {
+
+      //[DocRef: Name=Picture Marker Symbol File-android, Category=Fundamentals, Topic=Symbols and Renderers]
+      //Create a picture marker symbol from a file on disk
+      BitmapDrawable pinBlankOrangeDrawable = (BitmapDrawable) Drawable.createFromPath(mPinBlankOrangeFilePath);
+      final PictureMarkerSymbol pinBlankOrangeSymbol = new PictureMarkerSymbol(pinBlankOrangeDrawable);
+      //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
```
Comment (doneill) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: @willcrick You should split this comment after the comma to the next line comment.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java [hunk 4]
```diff
@@ -108,7 +105,8 @@ public void run() {
     pinStarBlueSymbol.setHeight(40);
     pinStarBlueSymbol.setWidth(40);
     //Optionally set the offset, to align the base of the symbol aligns with the point geometry
-    pinStarBlueSymbol.setOffsetY(11); //The image used for the symbol has a transparent buffer around it, so the offset is not simply height/2
+    pinStarBlueSymbol.setOffsetY(
+        11); //The image used for the symbol has a transparent buffer around it, so the offset is not simply height/2
```
Comment (doneill) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: @willcrick Can you move this comment above the method call, then you won't need to return and reads better?

**Comment by doneill** [comment]
@willcrick Looks good, just a few minor comments and we are good to merge.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java (hunk 1 above)
Comment (willcrick) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: updated.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java (hunk 2 above)
Comment (willcrick) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: updated.

File: picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java (hunk 3 above)
Comment (willcrick) [inline picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java]: updated.

**Comment by willcrick** [comment]
thanks @doneill have pushed updates for your comments.

**Pull request merged**